ldotel/go.mod

```diff
@@ -0,0 +1,35 @@
+module ldotel
+
+go 1.21
+
+require (
+	github.com/launchdarkly/go-sdk-common/v3 v3.1.0
+	github.com/launchdarkly/go-server-sdk/v7 v7.2.0
+	github.com/stretchr/testify v1.8.4
+	go.opentelemetry.io/otel v1.24.0
```

Comment:
Is there any strategy we should use for versions of these dependencies. It seems like go would really like it to be the latest.

Reply:
Latest